stub/src/test/java/io/grpc/kotlin/ServerCallsTest.kt

```diff
@@ -357,7 +356,7 @@ class ServerCallsTest : AbstractCallsTest() {
     val responseChannel = Channel<HelloReply>()
     clientCall.start(object: ClientCall.Listener<HelloReply>() {
       override fun onMessage(message: HelloReply) {
-        responseChannel.sendBlocking(message)
+        responseChannel.trySendBlocking(message)
```

Comment:
I think `sendBlocking()` throws on failure so to preserve that this would need to be.

```kotlin
        responseChannel.trySendBlocking(message).getOrThrow()
```

Reply:
This is test code, and we don't expect this to throw in any event.